test/parser/controlFlow/If.test.js

```diff
@@ -165,6 +165,20 @@ describe("parser if statements", () => {
             expect(statements).not.toBeNull();
             expect(statements).toMatchSnapshot();
         });
+
+        it("allows multiple statements in 'then' block (#481)", () => {
```

Comment:
aaaaand the unit test.  Everything else is a snapshot change caused by the corrected off-by-one in location tracking.